commands/remind.js

```diff
@@ -32,20 +32,31 @@ module.exports = {
 		}
 
 		const endTime = startTime + userInputTime;
-		idToRemove = createReminder.run(message.author.id, message.channel.id, userReminder, startTime, endTime).lastInsertRowid;
+		let guild_id;
+		if (message.channel.type == "dm") {
+			guild_id = "dm";
+		} else {
+			guild_id = message.guild.id;
+		}
```

Comment:
```suggestion
		let guild_id = message.guild.id;
		if (message.channel.type === "dm") {
			guild_id = "dm";
		}
```
this could be simplified

Reply:
Thanks! Although, we need to default to `dm`, otherwise this fails when we set up a reminder in a DM.